nectar/sample-config.toml

```diff
@@ -23,7 +23,13 @@ dai = 1000
 [maker.maximum_possible_fee]
 # An estimation of the maximum fee that we would expect to pay, used to ensure we always have enough
 # balance to execute an order we publish.
-bitcoin = 0.00009275 
+bitcoin = 0.00009275
+
+# Strategies used for Bitcoin fee handling.
+[maker.fee_strategies.bitcoin]
+# A static sat/byte value can be specified or you can relay on bitcoind's estimatesmartfee feature.
+strategy = "static"
+sats_per_bytes = 53
```

Comment:
Naming nit :)

```suggestion
sats_per_byte = 53
```

Reply:
Also, should we use a better value here? :D

53 seems a lot!

Reply:
Hum, it's a sample file, not default, but happy to change.